web/migrations/0050_achievement_badge_type_achievement_teacher_and_more.py

```
@@ -0,0 +1,69 @@
+                ),
+                ("token_id", models.CharField(blank=True, max_length=100)),
+                ("contract_address", models.CharField(blank=True, max_length=100)),
+                ("transaction_hash", models.CharField(blank=True, max_length=100)),
+                ("metadata_uri", models.URLField(blank=True)),
+                ("minted_at", models.DateTimeField(blank=True)),
+                ("wallet_address", models.CharField(blank=True, max_length=100)),
+                ("icon_url", models.URLField(blank=True)),
+                (
+                    "achievement",
+                    models.OneToOneField(
+                        on_delete=django.db.models.deletion.CASCADE, related_name="nft_badge", to="web.achievement"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)